### PR TITLE
Add link to appointment booking api and clear some empty pages.

### PR DESCRIPTION
**Pull request by afzalmufti**
Small change, commenting out menu links to some empty pages pending new content being developed.

Staging link: https://developer-test.nhs.uk/apis/uec-tech-standards/appointments.html

### CODE REVIEW

**Comment by mattstibbs** [comment]
Thanks @afzalmufti 

Could you update the link to appointments documentation to be **https://developer.nhs.uk/apis/uec-appointments** please

File: _pages/emerging/appointments.md [hunk 1]
```diff
@@ -6,4 +6,7 @@ toc: false
 folder: emerging
 ---
 
-{% include note-notpublished.html %}
+The UEC appointment booking standard is an open standard supporting booking across many care settings.
+It utilises the CareConnect FHIR messaging standard and is being published in the form of a website.
+
+[Access the appointment booking API documentation](https://nhsd-a2si.github.io/docs-uec-appts/)
```
Comment (mattstibbs) [inline _pages/emerging/appointments.md]: Please update this link to https://developer.nhs.uk/apis/uec-appointments/

**Review by mattstibbs — CHANGES_REQUESTED** [review]
Please update appointments URL as per comment

**Comment by afzalmufti** [comment]
URL amended, should be good to go now. Thanks!

**Review by mattstibbs — APPROVED** [review]

**Pull request merged**